dist/gmusic.js

```diff
@@ -207,6 +244,11 @@ proto.playback = {
       return GMusic.Playback.NO_SHUFFLE;
     }
   },
+  setShuffle: function (mode) {
+    while (this.playback.getShuffle() !== mode) {
```

Comment:
This could turn bad quickly if the user manually inputs and invalid `mode`.

If they put `mode = "yolo"` this will be an infinite loop :laughing: 

We should validate the `mode` variable against our ENUM